browser/test/unit/peopleStore.js

```diff
@@ -86,11 +95,25 @@ describe('PeopleStore', function() {
 		global.E2.app.dispatcher.dispatch = function(pl) {
 			assert.equal(pl.actionType, 'uiMouseMoved')
 			assert.equal(pl.x, 32)
-			assert.equal(pl.y, 64)
+			assert.equal(pl.y, 32)
+			done()
+		}
+
+		ps._mouseMoveHandler.call(ps, { pageX: 64, pageY: 64 })
+	})
+
+	it('does not dispatch on mousemove when not over the canvas', function(done) {
+
+		this.timeout(testTimeout + 500)
```

Comment:
Please don't use timers in tests, they could be wildly unreliable eg. in a heavily loaded server. :)

Maybe instead throw an error if you *do* get a dispatch (would happen immediately, in the same call stack, anyway) and otherwise just exit